### DISCUSSION TITLE
repeated properties in an object

### DISCUSSION THREAD

**Discussion by arademaker** [json-schema-org/community#777] [Q&A]
now to check for repeated properties in an object? I want to avoid to "value" in 

```
{ "description": "..." , "value": 1, "value": 1}
```

**Comment by Julian** [comment]
JSON Schema is built (at least historically) on JSON, and repeating a key in JSON is actually undefined (parser-specific behavior). So JSON Schema doesn't really have a way to express that because it isn't even universally describable as JSON.

If you have such JSON and are trying to prevent it I think the most obvious place to look would possibly be finding a JSON linter which prevents it.
